cpp/CMakeLists.txt

```diff
@@ -375,6 +375,13 @@ elseif(DOLFINX_ENABLE_ADIOS2)
   find_package(ADIOS2 2.8.1)
```

Comment:
might want to bump to ADIOS2 2.10.0 for `adios2.is_built_with_mpi` (see below)
I suggest not pushing this into dolfinx 0.8 since ADIOS2 2.10 has only just been released.

Reply:
We only require this in one test in Python, so we can keep the lower limit (2.8.1) in C++.